docs/interpreter_checklist.md

```diff
@@ -0,0 +1,67 @@
+# StableHLO Interpreter Checklist
+
+In this document, we summarize the guidelines while implementing and reviewing
+interpreter for an op. We have intentionally tied a few auxiliary action items,
+related to verifier and type inference, with the idea of making progress in
+those fronts alongside the interpreter implementation.
```

Comment:
What about ops that have already been implemented?

Reply:
All the ops which has the interpreter will be assigned (to me) and checked for compliance with this checklist.

Reply:
Here is the ticket for the assignment of those ops
 https://github.com/openxla/stablehlo/issues/1002